tests/libYARP_sig/ImageTest.cpp

```diff
@@ -226,7 +226,6 @@ class ImageTest : public UnitTest {
                         yAssert(data);
                         img.setQuantum(pad1);
                         img.setPixelCode(VOCAB_PIXEL_RGB);
-                        img.setPixelSize(3);
```

Comment:
This should still work after your changes, if it is not working you are doing something wrong.
Please restore this line in the unit test...